### DISCUSSION TITLE
Skywalking-python not get the metric by Django+uwsgi

### DISCUSSION THREAD

**Discussion by katelei6** [apache/skywalking#8569] [Q&A]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

Python Agent (apache/skywalking-python)

### What happened

when I use run manager.py， the ui can get the metric.
when I use uwsgi to run it, the ui can not get the endpoint.
<img width="1607" alt="image" src="https://user-images.githubusercontent.com/79354895/155101925-2cc089d5-53aa-4981-add6-fb0af32aa3e7.png">

wsgi.py
from __future__ import absolute_import
import os

from django.core.wsgi import get_wsgi_application

from skywalking import agent, config
config.init(collector_address='127.0.0.1:12800', service_name='test')
agent.start()

os.environ.setdefault("DJANGO_SETTINGS_MODULE", "jcdp.settings")

application = get_wsgi_application()

### What you expected to happen

I think in the ui ,we can get the metric, the endpoint,
but actually, there is noting.

### How to reproduce

1. pip install apache-skywalking
2. edit the wsgi.py
from __future__ import absolute_import
import os

from django.core.wsgi import get_wsgi_application

from skywalking import agent, config
config.init(collector_address='127.0.0.1:12800', service_name='test')
agent.start()

os.environ.setdefault("DJANGO_SETTINGS_MODULE", "jcdp.settings")

application = get_wsgi_application()
3. run the server

### Anything else

I see the other issue.

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
I am not sure why you submitted this as a bug directly, as in your description, can't see any error. You have some traces doesn't mean you will have metrics.

**Comment by katelei6** [comment]
I send some request，but not get the endpoint.
when I use manage.py to run , I got the info:
<img width="591" alt="image" src="https://user-images.githubusercontent.com/79354895/155105007-3c7fe625-645e-4b63-aba1-468b90c0c124.png">
but use uwsgi, nothing to get it.

**Reply by wu-sheng** [reply]
Endpoint usually needs entry spans. Do you have that? If not, check the agent interceptor.
I don't mean everything is good, but your descriptions don't make sense.

**Comment by katelei6** [comment]
The file i write the skywalking start is not right？@kezhenxu94

**Reply by kezhenxu94** [reply]
> The file i write the skywalking start is not right？@kezhenxu94

From the screenshot you provided and the code snippet, they don't match up. Please provide a complete, runnable and reproducible demo, because we have other users proving under some cases Django+uwsgi works, but under some other cases it doesn't work (yet), so please provide full reproducible example. this is not runnable and reproducible 
<img width="682" alt="image" src="https://user-images.githubusercontent.com/15965696/155141317-38e72b4f-de82-4215-b0bf-b242e09886f2.png">
